exchange/exchange.go

```diff
@@ -316,8 +316,17 @@ func (e *exchange) HoldAuction(ctx context.Context, r *AuctionRequest, debugLog
 
 	recordImpMetrics(r.BidRequestWrapper, e.me)
 
+	// Retrieve host and account-level EEA countries config
+	eeaCountries := SelectEEACountries(e.privacyConfig.GDPR.EEACountries, r.Account.EEACountries)
```

Comment:
Nitpick: The comment is not necessary. The inputs are clearly host and account level, but if you'd like to make that association clearer consider a revised method name.